lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -162,7 +172,7 @@ class GoogleCloudOutput < BufferedOutput
     attr_reader :running_on_managed_vm
     attr_reader :gae_backend_name
     attr_reader :gae_backend_version
-    attr_reader :service_name
+    attr_reader :resource_type
```

Comment:
Ran into some weird issue when implementing this. A bug has been filed with Protobuf component to have it fixed.

Meanwhile, I've modified the implementation to store the non-grpc version of `MonitoredResource` and convert it to the grpc version in the end. A `TODO` has been added to construct the grpc version of `MonitoredResource` from the beginning after the protobuf bug is fixed.